shared/oae/css/oae.base.css

```diff
@@ -897,3 +897,31 @@ ul.as-list li.as-result-item.active {
 .oae-markdown-embedded p:first-child {
     margin-top: 0;
 }
+
+/**
+ * Markdown table styles
```

Comment:
- The header row has a grey background, so the `transparent` property isn't working
- There's no padding enforced between the table headers
- The first row under the table header should be white, only the one after that should be grey